ukis_pysat/files.json

```diff
@@ -0,0 +1,113 @@
+{
+ "LC08": [
```

Comment:
consider the use of enums for platform specific band attributes. connected to #23

Reply:
or [pydantic](https://pydantic-docs.helpmanual.io/)?

Reply:
you could try something like this (not sure if this works though):
```python
class Platform(Enum):
     Sentinel1 = "Sentinel-1"
     Sentinel2 = "Sentinel-2"

     @property
     def bands(self):
         if self == Platform.Sentinel1:
             return [
                 {"id": 0, "name": "B1", "wavelength": "blue"},
                 {"id": 1, "name": "B2", "wavelength": "green"},
             ]
         elif self == Platform.Sentinel2:
             return [
                 {"id": 0, "name": "B010", "wavelength": "blue"},
                 {"id": 1, "name": "B020", "wavelength": "green"},
             ]

print(Platform.Sentinel1.bands)
```